{"additions": 165, "auther_ref": "fix27882", "auther_sha": "7dcc3164f6b0c43c54aeee5e7e710bafe1ac926d", "author": "fabiocos", "body": "#### PR description:\r\n\r\nRestore #27882, revert #28032\r\n\r\n#### PR validation:\r\n\r\nSee #28041 by @mbluj \r\nSuccessfully tested with ```runTheMatrix.py -l 1325.51,1329.1,136.772,136.8521 -i all --ibeos```", "branch": "CMSSW_10_6_X", "changed_files": 13, "comments": 3, "commits": 2, "created_at": "1569080788", "deletions": 42, "labels": ["analysis-pending", "comparison-pending", "operations-pending", "orp-pending", "pending-signatures", "reconstruction-pending", "tests-started", "xpog-pending"], "milestone": "CMSSW_10_6_X", "number": 28046, "release-notes": [], "review_comments": 0, "state": "open", "title": "Restore Disabled PCA inputs for deepTau v2 (10_6_X)", "updated_at": "1569080932", "user": "fabiocos"}